src/app/directives/restrict-double-quote.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, HostListener } from '@angular/core';
+
+@Directive({
+  selector: '[appRestrictDoubleQuote]'
+})
+export class RestrictDoubleQuoteDirective {
+
+  constructor() { }
```

Comment:
@Input('chars') chars: string  = "34"; // a comma separated list of special char code values 

... and default is 34